lang/java/tools/pom.xml

```diff
@@ -221,11 +221,6 @@
       <version>${commons-logging.version}</version>
     </dependency>
     <dependency>
-      <groupId>commons-httpclient</groupId>
```

Comment:
Yes, it turned out we do not require this dependency neither for compile nor for the unit tests. It might cause a backward incompatible change if a consumer relies on that this package is among its transitive dependencies but the whole change would be in 1.9 only so I shall not be a problem.